traits.html

```diff
@@ -1,29 +1,35 @@
 <!DOCTYPE html>
 <html>
+
 <head>
   <meta charset="UTF-8">
   <title>Traits</title>
   <link rel="stylesheet" href="traits.css">
 </head>
 
-<body>
+<center>
+  <body>
+    <section class="traitsContainer" id="traitsContainer">
+      <div class="" id="">
+        <h2>Here are the results</h2>
+      </div>
+    </section>
+</center>
+<br>
+<center>
   <div class="traitsSidebar">
+    <img class="image" src="./images/info.png">
```

Comment:
it's repetitive to make a class called 'image' if applied to all img. Consider removing or changing?